team-telnyx/webrtc
# Refactor client and call interfaces

- Differentiate between exported SIP client and SDK client interfaces
  - Renamed `IClientOptions` -> `ISIPClientOptions`
  - Renamed `ITelnyxRTCOptions` -> `IClientOptions`
  - Differentiated internal call options from verto call options by declaring `IVertoCallOptions`
- Move option param documentation to public interface

Followed by https://github.com/team-telnyx/react-developer-docs/pull/310/files

## 📝 To Do


## ✋ Manual testing

1. Run `npm run docs` in `packages/js`
2. Preview markdown files. Verify that `IClientOptions` and `ICallOptions` are now documented
3. Run `react-client` example. Verify app works as expected

## 📸 Screenshots

<img width="406" alt="Screen Shot 2021-01-14 at 1 07 34 PM" src="https://user-images.githubusercontent.com/4672952/104649478-a681a080-5669-11eb-947a-b2539e448ee6.png">
